tck/src/main/java/ee/jakarta/tck/data/standalone/persistence/PersistenceEntityTests.java

```diff
@@ -110,8 +110,8 @@ public void testEntityManager() {
         catalog.save(Product.of("socks", 5.99, "TEST-PROD-87", Department.CLOTHING));
         catalog.save(Product.of("volleyball", 10.99, "TEST-PROD-89", Department.SPORTING_GOODS));
 
-        assertEquals(385.95, catalog.sumPrices(Department.CLOTHING, Department.SPORTING_GOODS), 0.001);
-        assertEquals(794.09, catalog.sumPrices(Department.CLOTHING, Department.APPLIANCES), 0.001);
+//        assertEquals(385.95, catalog.sumPrices(Department.CLOTHING, Department.SPORTING_GOODS), 0.001);
```

Comment:
```suggestion
        boolean isSupported;
        try {
            assertNotNull(catalog.getEntityManager());
            isSupported = true;
        } catch (UnsupportedOperationException x) {
            isSupported = false;
        }

        if (isSupported) {
            assertEquals(385.95, catalog.sumPrices(Department.CLOTHING, Department.SPORTING_GOODS), 0.001);
```

Reply:
This suggestion assumes that the provider is not implemented to reject the code at compilation time. But the spec explicitly allows the provider to do that.